### DISCUSSION TITLE
Screenshots come up blank when brightness dimmed

### DISCUSSION THREAD

**Discussion by fxerkan** [MonitorControl/MonitorControl#1453] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

Hi,
I'm using Macbook Air 2020 M1 chipset with 2 Extended Displays (Dell S2721QS & Dell P2719H) via a Dock station (plugable displaylink) so that there is no brightness adjustment option. 

Because of that I installed MotionControl 4.2.0 app to control the extended displays brightness at software level dimming and  hardware level not available at my monitors.

When I set the brightness level %100 there is no issue, I can take screenshots with any of these applications at any displays.
* Apple's own screenshot functionality ( cmd+shift+ 4 or 5 )
* Parallels Toolbox Screen Capture 
* Lightshot screenshot ( cmd+shift+s )

But, when I decreased the brightness under the %100 like %90, %80, %70..  and try to take a screenshot at any extended display - all the screenshots seems blank or black only color.

If i set the brightness again at %100 everything works as expected.

### Steps to reproduce

1. use an extended display which does not support hardware dimming
2. decrease the brightness %90 or %75
3. try to take a screenshot at the extended display with one of these tools (Apple's own screenshot functionality , Parallels Toolbox Screen Capture, Lightshot screenshot 
4. check the saved screenshot is blank/black
5. 

### Expected behavior

saved screenshot should be like this, not blank or black only
<img width="1280" alt="100 Brightness Screenshot" src="https://github.com/MonitorControl/MonitorControl/assets/20793273/39f49e26-277e-48f2-ae78-2dda2ab39b39">

### Anything else?

Issue %90 brightness-Apples Screenshot as you can see its a blank image
<img width="2560" alt="90 brightness-Apples Screenshot" src="https://github.com/MonitorControl/MonitorControl/assets/20793273/9545a7db-c22a-4e99-9d46-6c766197cdc9">

%90 brightness-Parallels Toolbox Capture Screen
<img width="640" alt="90 brightness-Parallels Toolbox Capture Screen" src="https://github.com/MonitorControl/MonitorControl/assets/20793273/67191e00-50b2-4cfa-89af-87f060f265fe">

%90 brightness-Lightshot
<img width="640" alt="90 brightness-Lightshot" src="https://github.com/MonitorControl/MonitorControl/assets/20793273/a9c3692f-af1c-445a-8f12-785cec12f478">

### Environment Information (please complete the following information)

```markdown
- macOS version: Sonoma 14.0 (23A344)
- Mac model: MacBook Air M1 2020
- MonitorControl version: 4.2.0 Build 7048
- Monitor(s): Dell S2721QS and Dell P2719H
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment, marked as answer]
This happens if only software dimming is applied and gamma table manipulation is disabled in the app or is unavailable for the display due to some incompatibility. In this case a black overlay is used with changing transparency as a last resort - the screenshot app captures this black overlay.

**Comment by forthrin** [comment]
+1

The screenshot is actually much darker (here) than the dimmed screen. Can this be resolved, or is it WONTFIX because of sandboxing / the way the macOS video pipeline works?
